src/nrdbhandler.cpp

```diff
@@ -128,10 +134,14 @@ bool
 NrBaseDbHandler::executeQuery(QSqlQuery &query)
 {
     bool ok = query.exec();
-    if(!ok) {        
-        *m_logger << UNQL::LOG_CRITICAL << "EXECUTE QUERY ERROR - got error: " << query.lastError().text() << " for query (" << query.lastQuery() << ")" << UNQL::eom;
+    if(ok) {
+
```

Comment:
Missing variable reset.

BTW, wouldn't be better to reset vars before execution and then check for errors only?
Something like:
```
resetError();
bool ok = doCommand();
if (!ok) {
  setError(errType, errString);
}
```
I would suggest to use private methods to set/unset errors, just to make it clear.

Reply:
Very well, I agree